gateway-provider-ha/src/main/java/org/apache/knox/gateway/ha/provider/HaServiceConfig.java

```diff
@@ -57,4 +57,8 @@
   boolean isNoFallbackEnabled();
 
   void setNoFallbackEnabled(boolean noFallbackEnabled);
+
+  void setDisableStickySessionForUserAgents(String disableStickySessionForUserAgents);
```

Comment:
(minor) nit: The method name is misleading to me. 
setStickySessionDisabledUserAgents() ?